Fix launch_xml examples

jacobperron opened the pull request:
Remove 'sep' attribute and use square brackets for lists.

As far as I know, this is the correct sytax for a list value and the 'sep' attribute was never implemented.

@hidmic @ivanpauno Does this look correct?

ivanpauno commented:
> As far as I know, this is the correct sytax for a list value and the 'sep' attribute was never implemented.

IIRC, you have to use `value="Some phrase,'100.0','true'" value-sep=","`

jacobperron committed:
Fix launch_xml examples

The separator attribute needs to be prefixed with the name of the attribute that it is affecting.

Signed-off-by: Jacob Perron <jacob@openrobotics.org>

jacobperron committed:
Add example without *-sep attribute

Signed-off-by: Jacob Perron <jacob@openrobotics.org>

jacobperron commented:
> IIRC, you have to use value="Some phrase,'100.0','true'" value-sep=","

Interesting. I found some docs about it [here](https://github.com/ros2/launch/tree/91536a96d699aeb2eb20b9f59e8ef7308255a46e/launch_xml).

I've updated this PR with the correct syntax, and also added an example that doesn't use the *-sep attribute.
